src/jdk.compiler/share/classes/com/sun/tools/sjavac/Util.java

```diff
@@ -79,15 +79,23 @@ public static String extractStringOption(String opName, String s) {
         return extractStringOption(opName, s, null);
     }
 
-    public static String extractStringOption(String opName, String s, String deflt) {
+    private static String extractStringOptionWithDelimiter(String opName, String s, String deflt, char delimiter) {
         int p = s.indexOf(opName+"=");
         if (p == -1) return deflt;
         p+=opName.length()+1;
-        int pe = s.indexOf(',', p);
+        int pe = s.indexOf(delimiter, p);
         if (pe == -1) pe = s.length();
         return s.substring(p, pe);
     }
 
+    public static String extractStringOption(String opName, String s, String deflt) {
+        return extractStringOptionWithDelimiter(opName, s, deflt, ',');
+    }
+
+    public static String extractStringOptionLine(String opName, String s, String deflt) {
+        return extractStringOptionWithDelimiter(opName, s, deflt, '\n').strip();
```

Comment:
Is '\n' going to be problematic due to differences in line endings on various platforms?

Reply:
The JDK project is limited to unix-style line endings only. No other styles are supported, and any attempt to use such will break the build (often early on).